app/code/Magento/Catalog/Controller/Adminhtml/Product/Save.php

```diff
@@ -117,6 +117,7 @@ public function execute()
                 $productId = $product->getEntityId();
                 $productAttributeSetId = $product->getAttributeSetId();
                 $productTypeId = $product->getTypeId();
+                $this->getRequest()->setParam('type', $productTypeId);
```

Comment:
This is explained here: https://github.com/magento/magento2/pull/8526#issuecomment-279990671 You should not have attributes applicable to configurable and not applicable to simple products as they are interchangeable.